### DISCUSSION TITLE
Roda 3.59.0 Released

### DISCUSSION THREAD

**Discussion by jeremyevans** [jeremyevans/roda#280] [General]
Roda 3.59.0 has been released!

# New Features

* An additional_render_engines plugin has been added, for considering
  multiple render engines for templates. If the template path does not
  exist for the default render engine, then each additional render
  engine will be checked, returning the first path that exists:

  ```ruby
  plugin :additional_render_engines, ['haml', 'str']
  ```

  This is similar to the additional_view_directories plugin added in
  3.53.0.  Both plugins can be used if you want to consider multiple
  view directories and multiple render engines.

# Other Improvements

* A typo in a private method name in the delete_empty_headers plugin
  has been fixed.

Thanks,
Jeremy

**Comment by kez** [comment]
Haml brings back a lot of memories.  Not all of them good!
